src/views/Receiver.vue

```diff
@@ -232,6 +239,7 @@ export default {
       isStream: false,
       isPlaying: false,
       statsVisible: false,
+      volume: window.localStorage.getItem('volume_num') || 0.5,
```

Comment:
Not sure how I feel about the localStorage key name `volume_num`, although after thinking about it, maybe it's nice that the `_num` suffix is there since localStorage only lets you store `String`'s, and it needs to be parsed back into a `Number` upon retrieval.

Thoughts @codysherman?

Reply:
We can just leave this as-is, for the reasoning I mentioned in the comment above.

Reply:
Let's just make the key `'volume'`.

Reply:
@pociust I agree with Sam